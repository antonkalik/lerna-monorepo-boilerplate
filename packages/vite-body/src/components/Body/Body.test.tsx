import { render } from "@testing-library/react";
import { Body } from ".";

jest.mock("vite-common", () => ({
  Button: () => <div>Button</div>,
}));

describe("<Body />", () => {
  it("should render component", () => {
    const { container } = render(
      <Body
        title="Title test"
        subTitle="subtitle test"
        message="message test"
      />
    );

    expect(container).toMatchSnapshot();
  });
});
