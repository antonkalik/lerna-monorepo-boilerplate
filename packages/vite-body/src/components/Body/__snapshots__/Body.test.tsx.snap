// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Body /> should render component 1`] = `
<div>
  <div>
    <h1>
      Title test
    </h1>
    <h2>
      subtitle test
    </h2>
    <p>
      message test
    </p>
    <div>
      Button
    </div>
  </div>
</div>
`;
